packages/contracts/contracts/core/component/MetaTxComponent.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@opengsn/contracts/src/BaseRelayRecipient.sol";
+
+import "./Component.sol";
+
+/// @title Base component in the Aragon DAO framework supporting meta transactions
+/// @author Michael Heuer - Aragon Association - 2022
+/// @notice Any component within the Aragon DAO framework using meta transactions has to inherit from this contract
+abstract contract MetaTxComponent is Component, BaseRelayRecipient {
+    /// @notice Role identifer to change the GSN forwarder
```

Comment:
is this notice is necessary?
if yes ? shouldn't we add to those that missing it ?

Reply:
No, I think it is self-explanatory, and we didn't add explanations for other member variables. I will remove it.